### PR TITLE
English language support

### PR DESCRIPTION
**Pull request by dixya**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     version=__version__,
     author='Pierre Nicolas Durette',
     author_email='pndurette@gmail.com',
-    url='https://github.com/pndurette/gTTS',
+    url='https://github.com/dixya/gTTS',
```
Comment (jhoelzl) [inline setup.py]: Haha, since this is a pull request you should not change the main repo url with our fork version.

**Comment by pndurette** [comment]
There's too many changes here I can accept (url, version), however I took your list into consideration, I'm going to test to see if there's any discernible difference. gTTS `2.0.0` will let you able to disable strict language tag checking, so you can still pass any of those (that Google might take). Except that soon.

**Comment by CoreyCole** [comment]
@pndurette did you find any discernible difference between the different types of english?

**Comment by CoreyCole** [comment]
If so, I made a simplified fork that only changes the langue codes (#105)

**Pull request closed without merging**